requirements-conda.txt

```diff
@@ -32,6 +32,5 @@ future==0.16.0
 matplotlib=1.5.3
 pysam=0.12.0.1
 pybedtools=0.7.10
-Jinja2=2.8
```

Comment:
This was moved because we are really only using it rendering the conda recipe?

Reply:
AFAICT, yeah, it isn't used for anything but building conda (which is why I moved it to that conditional line with the superfluous semicolon).